Documentation/sourcebuild-in-repos/update-dependencies.md

```diff
@@ -0,0 +1,28 @@
+# Updating a dependency
+
+In most cases, dependency flow should automatically update the packages you depend on.  There are some special considerations and things to keep in mind when doing manual updates.
+
+## External packages
```

Comment:
Consider inverting the order so that _Internal packages_ comes first.  This aligns with new-dependency doc and puts the most frequent type of update first.